.ruby-version

```diff
@@ -0,0 +1 @@
+2.7.2
```

Comment:
Controversial

Reply:
Woops - must have slipped in there after I upgraded
https://github.com/DFE-Digital/teacher-training-api/commit/ecb382333e3d38b75ccca4377b3fe035e4c26f9f